src/js/controller/controller.js

```diff
@@ -565,7 +565,9 @@ Object.assign(Controller.prototype, {
                     // Force tags to prime if we're about to play an ad
                     // Resetting the source in order to prime is OK since we'll be switching it anyway
                     if (inInteraction() && !_backgroundLoading) {
-                        _model.get('mediaElement').load();
+                        const video = _model.get('mediaElement');
+                        video.src = '';
+                        video.load();
```

Comment:
This is going to wipe out preloading on iOS where `backgroundLoading` is disabled.

When reproducing the bug do we have an instream instance here? If so, checking for that before clearing the source would prevent us from breaking preloading and only clear the source when there is a blocking instream instance.

Reply:
Thank you for the feedback!
We have an instream instance here when we are about to play an ad, and added the check to prevent breaking preloading.